face_api_dataset/modality.py

```diff
@@ -250,17 +250,27 @@ class Modality(Enum):
     
     **Type**: `int`.
     """
-    LANDMARKS_DENSE_MEDIAPIPE = auto()
+    LANDMARKS_3D_MEDIAPIPE_FACE = auto()
     """
     MediaPipe dense landmarks in 3D. Each landmark is given by three coordinates (x,y,z) in camera space.
 
     **Type**: `ndarray[float32]`. **Shape**: `(468, 3)`.
     """
-    
-    LANDMARKS_DENSE_SAI = auto()
+    LANDMARKS_3D_SAI = auto()
     """
-    SAI dense landmarks in 3D. Each landmark is given by three coordinates (x,y,z) in camera space.
+    SAI dense face landmarks in 3D. Each landmark is given by three coordinates (x,y,z) in camera space.
```

Comment:
Are you sure about comments here? Looks like it was related to LANDMARKS_3D_MEDIAPIPE_FACE. Could you double check it?

Reply:
Don't see any problem with the comment. What exactly is wrong?
SAI dense are face landmarks in our own format.